#
#	UsageEnviroment's destructor is empty/protected/not accessible.
#	reclaim() is the only possibility to destruct object.
#
#	demux_rtp.cpp |    2 +-
#	1 files changed, 1 insertion(+), 1 deletion(-)
#
--- MPlayer-1.0pre5try2/libmpdemux/demux_rtp.cpp.orig	2004-06-03 15:30:55.000000000 +0200
+++ MPlayer-1.0pre5try2/libmpdemux/demux_rtp.cpp	2004-12-17 17:23:59.866437376 +0100
@@ -407,7 +407,7 @@
   delete rtpState->sdpDescription;
   delete rtpState;
 
-  delete env; delete scheduler;
+  env->reclaim(); delete scheduler;
 }
 
 ////////// Extra routines that help implement the above interface functions:
